test/assert.js

```diff
@@ -2251,26 +2251,33 @@ describe('assert', function () {
         heroes = ['spiderman', 'superman'],
         fn     = function() { obj.value += 5 },
         fnDec  = function() { obj.value -= 20 },
+        getterFn = function() { return obj.value },
+        getter= function() {},
         bangFn = function() { obj.str += '!' },
         smFn   = function() { 'foo' + 'bar' },
         batFn  = function() { heroes.push('batman') },
         lenFn  = function() { return heroes.length };
 
+    getter.value= obj.value;
     assert.changes(fn, obj, 'value');
+    assert.changes(fn, getterFn, 'value');
     assert.changesBy(fn, obj, 'value', 5);
     assert.changesBy(fn, obj, 'value', -5);
+    assert.changesBy(fn, getterFn, 5);
```

Comment:
Please take a look at the comment I've made on the first assertion test because that is valid to all additions below, such as this one.

Reply:
> so basically resets all the feature changes right?

Yes, sir.

> Let's make sure all the code is correctly indented too.

Agreed.

---
Sorry that i have made unnecessary notification and work for you. I already take a look at your slack status about 5-10 min, because i already had bad experience with `git rebase`. So i ask Keith instead, but still i [get lost](https://slack-files.com/T0901SDRC-F84725QE4-b55bdbbf46) on my own confusion. With your aid i found some [clarity](https://runkit.com/adi/5a16d0989cd06300128397b2), but it seems i found it while you do the last review.

Due to unexpected rain, i made those dirty line. I have to save really quick and forgot to clean up, i will make sure it won't happen again. I will watch video that you mention, when i get lost again about `git rebase`.1. I am also not sure are you still busy on real world or not 2. Based on previous response time, i assume that this PR will get a bit of delay. Somehow i miss your last review on this PR. 

I really felt bad about this, hope on tomorrow. It should bring fresh state.

Reply:
Hey buddy, there's no need to feel bad about it. We're very happy that you want to contribute. Please don't worry about that 😃 

Basically, when doing a `rebase -i` you will be prompted with a list of commits and a few actions to do with them.

Whenever you want to drop a commit you can just use the `drop` word in front of it.

In order to ammend a commit into the last one you can use `fixup` and you can also use `squash` to ammend commits and change their messages as well.

After doing it a few times you will see it's simple, you just gotta try, it's totally normal to make mistakes the first times you try it.